src/math/hyp1f1.hh

```diff
@@ -194,7 +194,7 @@ namespace rascal {
         } else {
           // TODO(mc) this could be done with telescopic sums - and might be
           // faster than checking for bailout condition
-          for (size_t i{0}; i < static_cast<size_t>(n_terms); ++i) {
+          for (int i{0}; i < n_terms; ++i) {
```

Comment:
How does this not throw an comparison warning? `n_terms` above is a size_t, or not?